charts/testkube-api/templates/_helpers.tpl

```diff
@@ -163,8 +163,6 @@ Define API environment in agent mode
 - name: TESTKUBE_PRO_MIGRATE
   value:  "{{ .Values.cloud.migrate }}"
 {{- end}}
-- name: "NATS_EMBEDDED"
```

Comment:
I cannot see the commit that removes this option to see the reasoning behind it.
Did this go straight to main?

Reply:
I'm merging dev to main. Suppose, that dev should contain the latest version. But might be, it was added only to main. We can wait @exu to confirm

Reply:
He is on vacation till the end of the week, but yes I think that is what happened.

Reply:
Let's wait then, might be the whole pr is just missed commits from main to dev